docs/pages/kubernetes-access/getting-started.mdx

```diff
@@ -189,47 +111,74 @@ one Kubernetes group named `viewers` and one Kubernetes user.
 1. Apply the `ClusterRoleBinding` with `kubectl`:
    
    ```code
-   $ tsh kubectl apply -f viewers-bind.yaml
+   $ kubectl apply -f viewers-bind.yaml
    ```
 
-1. Log out of Teleport and log in again.
+## Step 2/3. Follow guided enrollment
 
-1. List connected clusters using `tsh kube ls`:
+To enroll a Kubernetes cluster using the Teleport Web UI:
 
-   ```code
-   $ tsh kube ls
-   ```
+1. Open the Teleport Web UI and sign in using your administrative account.
 
-   The command displays current Kubernetes clusters.
+1. Click **Enroll New Resource**.
 
-   ```
-   tsh kube ls
-   Kube Cluster Name Labels Selected 
-   ----------------- ------ -------- 
-   example-minikube        *        
-   ```
-   
-   If you have more than one cluster enrolled, you can switch between clusters 
-   by running a `tsh kube login <Var name="cluster-name"/>` command.
-  
-1. View pods using the `kubectl` command routed through the Teleport cluster:
+1. Type all or part of **Kubernetes** in the Search field to filter the resource types 
+   displayed, then click **Kubernetes**.
+
+1. Copy the command to add the `teleport-agent` chart repository and paste 
+   it in a terminal on your workstation.
+
+1. Type `teleport-agent` for namespace where you will deploy the Teleport
+   Kubernetes Service and the display name to use when connecting to this
+   cluster, then click **Next**.
+
+   After you click Next, Teleport generates a script to configure and enroll the 
+   Kubernetes cluster as a resource in the Teleport cluster.
+
+1. Copy the command displayed in the Teleport Web UI and run it in your
+   terminal.
+
+   The Teleport Web UI displays "Successfully detected your new Kubernetes cluster" as 
+   confirmation that your cluster is enrolled. When you see this message, click **Next**
+   to continue.
+
+## Step 3/3. Test Kubernetes access
+
+You can now set up access for specific Kubernetes groups and users to test access to
+the Kubernetes cluster you just enrolled.
+
+If you followed the previous steps in this guide, the **Set Up Access** view
+populates the **Kubernetes Groups** field with `viewers`.
+
+To set up and test access:
+
+1. Click **Next**.
+
+1. Specify the `teleport-agent` namespace, the Kubernetes `viewers` group from
+   the previous step, and your Teleport user name.
+
+1. Copy and run the commands displayed in the Teleport Web UI to interact with the
+   Kubernetes cluster and verify access through Teleport:
    
    ```code
-   $ tsh kubectl get pods 
+   $ tsh login --proxy=<Var name="teleport.example.com"/>:443 --auth=local --user=<Var name="admin@example.com"/> <Var name="teleport.example.com"/>
+   $ tsh kube login <Var name="Kubernetes-cluster-name"/>
```

Comment:
Should we consider adding an intermediate step here to introduce `tsh kube ls` and `tsh kube login` commands?

This getting start guide assumes too much and feels like if you run all this it will be fine but from the user perspective, going through the getting start guide won't provide the knowledge required to make it work if something else changes or to understand the concepts

Reply:
I've added context throughout the guide